src/api/script/delayed_job_monitor.rb

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby.ruby2.4
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'config', 'environment'))
+
+output = `service obsapidelayed status | grep "active"`
+
+if output.blank?
+  raise StandardError "obsapidelayed service is down!"
```

Comment:
This would trigger errbit then, right?

Reply:
and how to disable this when you want to take delayed jobs down for maintenance work?

Reply:
AFAIK we only ever restart delayed jobs? So at most that would be one error from a deploy, in which case I would say there's no point in disabling this check, we can just clear the error from errbit if we know its from a deploy.

Reply:
And yes @ChrisBr this will trigger errbit, thats the whole reason for raising this error.